pkg/v1/tkg/tkgctl/create_cluster.go

```diff
@@ -101,7 +101,7 @@ func (t *tkgctl) CreateCluster(cc CreateClusterOptions) error {
 		// For TKGS kubernetesVersion will be same as TkrVersion
 		options.KubernetesVersion = cc.TkrVersion
 		options.TKRVersion = cc.TkrVersion
-		err = confirmPacificKubernetesVersion(cc.SkipPrompt, options.KubernetesVersion)
+		//err = confirmPacificKubernetesVersion(cc.SkipPrompt, options.KubernetesVersion)
 		if err != nil {
 			return errors.Wrap(err, "unable to determine the kubernetes version for the cluster to be created on vSphere with Tanzu")
 		}
```

Comment:
What's the reason behind removing this check? Is it that we would like to just rely on feature-flag are we are removing the Version Check?

Also, if we are removing we should remove L-105-107 as well.

Reply:
it's by mistake, while testing code I did comment it, but I didn't really mean it. reverted changes.